learning/tour-of-beam/frontend/lib/cache/sdk.dart

```diff
@@ -19,30 +19,30 @@
 import 'dart:async';
 
 import 'package:flutter/widgets.dart';
+import 'package:playground_components/playground_components.dart';
 
-import '../models/sdk.dart';
 import '../repositories/client/client.dart';
 import '../repositories/models/get_sdks_response.dart';
 
 class SdkCache extends ChangeNotifier {
   final TobClient client;
 
-  final _sdks = <SdkModel>[];
+  final _sdks = <Sdk>[];
```

Comment:
We made a separate `SdkModel` here to be received from the ToB backend, and later learned we can reuse `Sdk` from the shared package by just adding one more deserialization to it.